src/js/components/DataChart/stories/StackedBars.js

```diff
@@ -18,28 +18,30 @@ const Example = () => (
     <Box align="center" justify="start" pad="large">
       <DataChart
         data={data}
+        series={[
+          {
+            property: 'date',
+            render: date => (
+              <Text margin={{ horizontal: 'xsmall' }}>
+                {new Date(date).toLocaleDateString('en-US', {
+                  month: 'numeric',
+                  day: 'numeric',
+                })}
+              </Text>
+            ),
+          },
+          'usage',
+          'bonus',
+        ]}
         chart={[
           {
-            keys: [
-              { key: 'usage', color: 'graph-1' },
-              { key: 'bonus', color: 'graph-2' },
-            ],
+            property: ['usage', 'bonus'],
             type: 'bar',
           },
         ]}
-        xAxis={{
-          key: 'date',
-          render: date => (
-            <Text margin={{ horizontal: 'xsmall' }}>
-              {new Date(date).toLocaleDateString('en-US', {
-                month: 'numeric',
-                day: 'numeric',
-              })}
-            </Text>
-          ),
-        }}
-        yAxis={{ guide: true }}
-        gap="medium"
+        axis={{ x: { property: 'date', granularity: 'fine' } }}
+        guide={{ y: true }}
+        legend
```

Comment:
Legend is unclear, I wasn't able to read it properly 
![image](https://user-images.githubusercontent.com/6320236/90075366-6ed98b80-dcba-11ea-8241-1b6234070cc5.png)

Reply:
oops, fixed.